docs/lib/codesandbox.ts

```diff
@@ -20,9 +21,9 @@ export const getCodeSandboxLink: (source: string) => string = (source) => {
 
   const usedComponents = Array.from(
     new Set(
-      (source.match(/<((\w+))|minorScale|majorScale|(((\w+)Icon))|Position/g) || []).map((component) =>
-        component.replace('<', '')
-      )
+      (
+        source.match(/<((\w+))|minorScale|majorScale|mergeTheme|defaultTheme|(((\w+)Icon))|Position/g) || []
```

Comment:
This will include `mergeTheme` and `defaultTheme` in the imports for a codesandbox link if the example needs it, which should prevent confusion for the live code examples in the doc site that "just work" and then have missing imports in CSB.